pkg/controller/rollout/rollout_progressing.go

```diff
@@ -149,10 +165,15 @@ func (r *RolloutReconciler) doProgressingInitializing(c *util.RolloutContext) (b
 		klog.Infof("verify rollout(%s/%s) TrafficRouting, and wait a moment", c.Rollout.Namespace, c.Rollout.Name)
 		return false, nil
 	}
+
+	// handler TrafficRouting
```

Comment:
plz comment about the fact that the traffic-routing here is a gateway that is not directly attach to the rolling application